src/components/ImageView/RegionView/RegionViewComponent.tsx

```diff
@@ -714,7 +714,7 @@ export class RegionViewComponent extends React.Component<RegionViewComponentProp
                     className={className}
                     width={this.props.width}
                     height={this.props.height}
-                    style={{left: this.props.left, top: this.props.top, cursor, border: `${this.props.highlighted ? "1px solid red" : "none"}`}}
+                    style={{left: this.props.left, top: this.props.top, cursor, border: `${this.props.highlighted ? "1px solid red" : "1px solid transparent"}`}}
```

Comment:
I wonder if this is causing additional displacement to the region canvas in non-channel-map mode:
<img width="581" alt="image" src="https://github.com/user-attachments/assets/c888233c-8c9e-46f7-ab9a-554ee4ace881" />

cf. dev
<img width="582" alt="image" src="https://github.com/user-attachments/assets/c30f2d60-d91b-49e3-ac14-4aa6c567e55f" />